skyfield/timelib.py

```diff
@@ -210,19 +210,21 @@ class Time(object):
     eps_correction = 0.0
 
     def __init__(self, ts, tt):
-        self.tt = tt
+        self.tt = _to_array(tt)
```

Comment:
Could you outline the use case you ran into that made this necessary? I'm not sure I've seen a Skyfield user ever instantiate `Time()` object directly — the library's design has them create `Time` objects through the methods of a `Timescale`, which I think has `_to_array()` everywhere it's needed. If one of them is missing, I'd prefer to fix it there rather than add a second call here.

Reply:
I did this so that `self.tt` could never be a python float, even if someone instantiates a Time object directly. If users never do that, then this wouldn't be necessary.